assets/prototype/domain/eventEditor/data/mutations/tickets/test/data.ts

```diff
@@ -0,0 +1,62 @@
+import { pickBy } from 'ramda';
+import { nodes as tickets } from '../../../queries/tickets/test/data';
+import { edge as priceEdge } from '../../../queries/prices/test/data';
+import { ReadQueryOptions } from '../../../queries/types';
+import { MutationInput, MutationType } from '../../../../../../application/services/apollo/mutations/types';
+import { ucFirst } from '../../../../../../application/utilities/text/changeCase';
+import { mutations } from '../../';
+
+const prices = { ...priceEdge, __typename: 'EspressoTicketPricesConnectionEdge' };
+
+export const mockedTickets = {
+	[MutationType.Create]: { ...tickets[0], id: tickets[0].id + '-alpha' }, // make sure to change the ID to make it different}
+	[MutationType.Update]: { ...tickets[0], prices },
+	[MutationType.Delete]: { id: tickets[0].id, __typename: tickets[0].__typename },
+};
+
+export const getMutationMocks = (mutationInput: MutationInput, mutationType: MutationType) => {
```

Comment:
maybe type return value too ?

Reply:
We need to have Apollo Query data defined for that. I will do that in a PR when I work on #2185